src/app/tests/suites/certification/Test_TC_BINFO_1_1.yaml

```diff
@@ -23,87 +23,70 @@ config:
-          [1653459580.065936][29174:29179] CHIP:EM: Sending Standalone Ack for MessageCounter:14961519 on exchange 31457i
-      disabled: true
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 0
+          constraints:
+              type: map32
 
     - label: "TH reads AttributeList from DUT"
-      verification: |
-          ./chip-tool basic read attribute-list 1 0
-
-          On TH, verify that DUT  sends a list of supported attributes
-
-          [1657021528.885544][7089:7094] CHIP:TOO: Endpoint: 0 Cluster: 0x0000_0028 Attribute 0x0000_FFFB DataVersion: 1869107664
-          [1657021528.885637][7089:7094] CHIP:TOO:   AttributeList: 25 entries
-          [1657021528.885669][7089:7094] CHIP:TOO:     [1]: 0
-          [1657021528.885695][7089:7094] CHIP:TOO:     [2]: 1
-          [1657021528.885720][7089:7094] CHIP:TOO:     [3]: 2
-          [1657021528.885744][7089:7094] CHIP:TOO:     [4]: 3
-          [1657021528.885768][7089:7094] CHIP:TOO:     [5]: 4
-          [1657021528.885793][7089:7094] CHIP:TOO:     [6]: 5
-          [1657021528.885817][7089:7094] CHIP:TOO:     [7]: 6
-          [1657021528.885841][7089:7094] CHIP:TOO:     [8]: 7
-          [1657021528.885865][7089:7094] CHIP:TOO:     [9]: 8
-          [1657021528.885890][7089:7094] CHIP:TOO:     [10]: 9
-          [1657021528.885915][7089:7094] CHIP:TOO:     [11]: 10
-          [1657021528.885939][7089:7094] CHIP:TOO:     [12]: 11
-          [1657021528.885964][7089:7094] CHIP:TOO:     [13]: 12
-          [1657021528.885988][7089:7094] CHIP:TOO:     [14]: 13
-          [1657021528.886013][7089:7094] CHIP:TOO:     [15]: 14
-          [1657021528.886037][7089:7094] CHIP:TOO:     [16]: 15
-          [1657021528.886062][7089:7094] CHIP:TOO:     [17]: 16
-          [1657021528.886086][7089:7094] CHIP:TOO:     [18]: 17
-          [1657021528.886111][7089:7094] CHIP:TOO:     [19]: 18
-          [1657021528.886135][7089:7094] CHIP:TOO:     [20]: 19
-          [1657021528.886160][7089:7094] CHIP:TOO:     [21]: 65528
-          [1657021528.886184][7089:7094] CHIP:TOO:     [22]: 65529
-          [1657021528.886209][7089:7094] CHIP:TOO:     [23]: 65531
-          [1657021528.886234][7089:7094] CHIP:TOO:     [24]: 65532
-          [1657021528.886258][7089:7094] CHIP:TOO:     [25]: 65533
-      disabled: true
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains:
+                  [
+                      65528,
+                      65529,
+                      65531,
+                      65532,
+                      65533,
+                      0,
+                      1,
+                      2,
+                      3,
+                      4,
+                      5,
+                      6,
+                      7,
+                      8,
+                      9,
+                      0xA,
```

Comment:
Looks like a more careful version of https://github.com/project-chip/connectedhomeip/pull/21289/files ?

Needs to regenerate darwin-framework-tool...

Reply:
ah, yeah - we've been attempting to get some of these things automated so you'll see a few of these. Let me close this one.